analyses/immune-deconv/02-summary-plots.R

```diff
@@ -29,6 +29,16 @@ deconvout <- opt$input
 output <- opt$output
 load(deconvout) 
 
+# if short histology is Medulloblastoma or ATRT, use them as broad histology
+deconv.res$broad_histology <- ifelse(deconv.res$short_histology %in% c("ATRT", "Medulloblastoma"),
+                                     deconv.res$short_histology, deconv.res$broad_histology)
+
+# add molecular subtype info
+deconv.res$broad_histology <- ifelse(is.na(deconv.res$molecular_subtype), 
+                                     deconv.res$broad_histology, 
+                                     paste0(deconv.res$broad_histology, '-', deconv.res$molecular_subtype))
+
+
```

Comment:
My preference would be to create a new variable/column (perhaps called something like `disease_labels`) that uses the logic you have here but doesn't replace `broad_histology`. The new column could then be used downstream. I think this is less confusing than someone reading `broad_histology` in your heatmap code and having the end result in the plot using "custom" labels. You will have to revisit this anyway (related to https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/627), so I think that change can wait.

Reply:
Hmm, makes sense. Thank you!